src/main/java/racinggame/business/Process.java

```diff
@@ -1,7 +1,21 @@
 package racinggame.business;
 
+import racinggame.model.*;
 import racinggame.response.GameResponse;
 
-public interface Process {
-    public GameResponse run(String[] names, int moves) ;
-}
+public class Process {
+
+    private String[] names;
+    private final Move move;
+
+    public Process(Move move, String[] names) {
+        this.names = names;
+        this.move = move;
+    }
+
+    public GameResponse run() {
+        JoinCars joinCars = CarFactory.createCars(names);
+        GameResult gameResult = move.moveCar(joinCars);
+        return new GameResponse(gameResult, gameResult.findLastRaceRecord().findGameWinners().WinnerCarsName());
```

Comment:
```suggestion
        return new GameResponse(gameResult, gameResult.findLastRaceRecord().findGameWinners().winnerCarsName());
```
메서드 이름은 소문자로 구현하는 것이 관례